ui/app/AppMain.qml

```diff
@@ -34,6 +34,19 @@ RowLayout {
         return popup
     }
 
+    function getUserNickname(pubKey) {
```

Comment:
getUserNickname is also used in CommunityMembersPopup and GroupInfoPopup, you'll need to point to appMain there too

Reply:
Good catch!!!